2025-04-08 20:27:12,572 INFO    MainThread:6580 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_setup.py:_flush():79] Configure stats pid to 6580
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_setup.py:_flush():79] Loading settings from /home/dazzy/.config/wandb/settings
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_setup.py:_flush():79] Loading settings from /home/dazzy/CLIP/wandb/settings
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'train_newloss.py', 'program_abspath': '/home/dazzy/CLIP/train_newloss.py', 'program': '/home/dazzy/CLIP/train_newloss.py'}
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_setup.py:_flush():79] Applying login settings: {}
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_init.py:_log_setup():533] Logging user logs to /home/dazzy/CLIP/wandb/run-20250408_202712-cmzk4sla/logs/debug.log
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_init.py:_log_setup():534] Logging internal logs to /home/dazzy/CLIP/wandb/run-20250408_202712-cmzk4sla/logs/debug-internal.log
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_init.py:init():619] calling init triggers
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {'EXP_ID': 0, 'EPOCH': 25, 'BATCH_SIZE': 50, 'LR_GB': 5e-05, 'LR_REST': 1e-08, 'LR_MIN': 1e-10, 'WEIGHT_DECAY': 0.1, 'BETA1': 0.98, 'BETA2': 0.9, 'EPS': 1e-06, 'EPOCH_LEN': 1916, 'WARMUP_T': 1900, 'WARMUP_MODE': 'auto', 'WARMUP_INIT_FACTOR': 1e-05, 'WARMUP_BY_EPOCH': False, 'LOG_DIR': '/home/dazzy/CLIP/Log', 'DEVICE': 'cuda', 'LOCK': True, 'LOCK_TEXT': False}
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_init.py:init():669] starting backend
2025-04-08 20:27:12,573 INFO    MainThread:6580 [wandb_init.py:init():673] sending inform_init request
2025-04-08 20:27:12,576 INFO    MainThread:6580 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-08 20:27:12,576 INFO    MainThread:6580 [wandb_init.py:init():686] backend started and connected
2025-04-08 20:27:12,580 INFO    MainThread:6580 [wandb_init.py:init():781] updated telemetry
2025-04-08 20:27:12,586 INFO    MainThread:6580 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2025-04-08 20:27:50,129 WARNING MainThread:6580 [wandb_init.py:init():1260] interrupted
Traceback (most recent call last):
  File "/home/dazzy/.conda/envs/CLIP/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1256, in init
    return wi.init()
           ^^^^^^^^^
  File "/home/dazzy/.conda/envs/CLIP/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 817, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dazzy/.conda/envs/CLIP/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 279, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dazzy/.conda/envs/CLIP/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dazzy/.conda/envs/CLIP/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 122, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dazzy/.conda/envs/CLIP/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dazzy/.conda/envs/CLIP/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2025-04-08 20:27:50,134 WARNING MsgRouterThr:6580 [router.py:message_loop():75] message_loop has been closed
